tools/models/metrics/run-scib-plots.py

```diff
@@ -0,0 +1,174 @@
+import sys
+import yaml
+import pickle
+import pandas as pd
+import plotnine as pt
+
+from typing import List, Dict
+
+
+def main():
+    try:
+        file = sys.argv[1]
+    except IndexError:
+        file = "scib-metrics-config-plot.yaml"
```

Comment:
@ebezzi Isn't there a `scib-metrics-config-plot.yaml` not yet checked in here, distinct from `scib-metrics-config.yaml` which is checked in?

Reply:
See https://github.com/chanzuckerberg/cellxgene-census/pull/945#issuecomment-2622837505

Reply:
@ebezzi This PR includes one YAML file `scib-metrics-config.yaml` which feeds into `run-scib.py`, with keys such as `census` `embeddings` `tissues`

Here we have a different script `run-scib-plots.py` which appears to expect a different YAML file named `scib-metrics-config-plot.yaml` with keys such as `metrics_files` `metrics_plots` `decimal_values` `normalize_entropy` etc. accessed below.

Did I miss something?